api_app/analyzers_manager/migrations/0071_analyzer_config_greynoise_labs.py

```diff
@@ -0,0 +1,135 @@
+from django.db import migrations
+from django.db.models.fields.related_descriptors import (
+    ForwardManyToOneDescriptor,
+    ForwardOneToOneDescriptor,
+    ManyToManyDescriptor,
+)
+
+plugin = {
+    "python_module": {
+        "health_check_schedule": None,
+        "update_schedule": {
+            "minute": "0",
+            "hour": "*",
```

Comment:
their docs says:
```
topC2s
Description
Return the top 1% of C2s ranked by pervasiveness GreyNoise has observed over the previous 24 hours. This data may be up to 4.5 hours old.
```

so we can reduce the number of update to once every 6 hours I'd say